packages/core/content-type-builder/server/src/controllers/components.ts

```diff
@@ -135,6 +138,10 @@ export default {
 
       const component = await componentService.deleteComponent(uid);
 
+      const migrationBuilder = new MigrationBuilder();
+      migrationBuilder.addDeleteComponent(uid);
+      await migrationBuilder.writeFiles();
```

Comment:
This isn't actually necessary anymore, but provides an example of how we could begin adding this feature everywhere for users who wish to use `database.settings.forceMigrations:false`